accumulo/src/main/java/org/vertexium/accumulo/AccumuloElement.java

```diff
@@ -153,6 +153,7 @@ protected void setVisibility(Visibility visibility) {
         );
 
         if (mutation.getNewElementVisibility() != null) {
+            getGraph().flush();
```

Comment:
Can you add a comment on this line describing why it's needed

Reply:
@joeferner Done with https://github.com/visallo/vertexium/pull/183/commits/3934f54bb4dd96a8300bb5a171a9d37e2cb9d9a4